src/app/components/mnoe-api/app-instances.svc.coffee

```diff
@@ -6,7 +6,7 @@ angular.module 'mnoEnterpriseAngular'
     @appInstances = []
 
     appInstancesPromise = null
-    @getAppInstances = ->
+    @getAppInstances = (force = false) ->
```

Comment:
The `force` is here to force unscoping?
In this case, it might more obvious to call it `unscoped`? And also add a quick comment on both `@getAppInstance` and `@fetchAppInstance`